bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestBotFactory.java

```diff
@@ -205,7 +205,16 @@ public List<Bot> create(BotConfiguration configuration) {
                 botBuilder.reviewCleanBackport(repo.value().get("reviewCleanBackport").asBoolean());
             }
             if (repo.value().contains("reviewMerge")) {
-                botBuilder.reviewMerge(repo.value().get("reviewMerge").asBoolean());
+                MergePullRequestReviewConfiguration result = MergePullRequestReviewConfiguration.JCHECK;
+
+                var val = repo.value().get("reviewMerge").asString().toLowerCase().trim();
+                if (val.equals("always")) {
+                    result = MergePullRequestReviewConfiguration.ALWAYS;
+                } else if (val.equals("never")) {
+                    result = MergePullRequestReviewConfiguration.NEVER;
+                }
```

Comment:
We should throw on an invalid configuration.

Reply:
Agree, fixed in latest commit.